determined-ai/determined
# fix: fix issue with db migrations

## Description
After a [PR](https://github.com/determined-ai/determined/pull/8093 ) got cherry-picked into the 0.26.1 release branch, it caused some db migration issues because it had a migration number after the earliest migration that was only in this release (allocation accelerators script). Fixed it by adding a CREATE TABLE IF NOT EXISTS to the script.




## Test Plan


1. checkout the git tag 0.26.1
2. start a fresh database
3. run devcluster on that tag
4. checkout main, run devcluster and it should not give you a db migration error


## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket
